content/manuals/subscription/change.md

```diff
@@ -0,0 +1,111 @@
+---
+description: Learn how to change your Docker subscription
+keywords: Docker Hub, upgrade, downgrade, subscription, Pro, Team, business, pricing plan
+title: Change your subscription
+aliases:
+- /docker-hub/upgrade/
+- /docker-hub/billing/upgrade/
+- /subscription/upgrade/
+- /subscription/downgrade/
+- /subscription/core-subscription/upgrade/
+- /subscription/core-subscription/downgrade/
+- /docker-hub/cancel-downgrade/
+- /docker-hub/billing/downgrade/
+weight: 30
+---
+
+{{< include "tax-compliance.md" >}}
+
+The following sections describe how to change plans when you have a Docker
+subscription plan or legacy Docker subscription plan.
+
+> [!NOTE]
+>
+> Legacy Docker plans apply to Docker subscribers who last purchased or renewed
+> their subscription before December 10, 2024. These subscribers will keep
+> their current plan and pricing until their next renewal date that falls on or
+> after December 10, 2024. To see purchase or renewal history, view your
+> [billing history](../billing/history.md). For more details about legacy
+> subscriptions, see [Announcing Upgraded Docker
+> Plans](https://www.docker.com/blog/november-2024-updated-plans-announcement/).
+
+## Upgrade your subscription
+
+When you upgrade from a free Personal subscription to a paid subscription, you immediately have access to all the features and entitlements available in your Docker subscription plan. For detailed information on features available in each subscription, see [Docker Pricing](https://www.docker.com/pricing).
+
+{{< tabs >}}
+{{< tab name="Docker subscription plan" >}}
+
+To upgrade your Docker subscription:
+
+1. Sign in to the [Docker Billing Console](https://app.docker.com/billing).
+2. Optional. If you're upgrading from a free user account to a Team subscription and want to keep your username, [convert your user account into an organization](../admin/convert-account.md).
```

Comment:
**issue:** There is a mix of different terminologies... 

Docker provides multiple account types `user`, `organization` and `company`. Where a user account can be converted to the organization but not vice versa.

Free/paid are applicable to Docker plans (e.g. Personal, Pro, Team or Business), but not accounts.

If a user account is subscribed to a paid plan (e.g. Docker Pro), then during account conversion, Docker Plan will be upgraded from Docker Pro to Docker Team.